plugins/none-ls/servers.nix

```
@@ -93,6 +93,9 @@ with lib; let
       staticcheck = {
         pacakge = pkgs.go-tools;
       };
+      ruff = {
+        package = pkgs.ruff;
+      };
```

```suggestion
      ruff = {
        package = pkgs.ruff;
      };
```
Please, preserve ordering (`staticcheck` should go below `ruff`)

---

Sorry I missed that! Fixed